charts/signoz/values.yaml

```diff
@@ -1518,6 +1529,10 @@ otelCollector:
           disk: {}
           filesystem: {}
           network: {}
+      httpreceiver/heroku:
+        # endpoint specifies the network interface and port which will receive data
+        endpoint: 0.0.0.0:8081
```

Comment:
@nityanandagohain this is `collectd`'s receiver default endpoint; please use something else.

Reply:
Ignoring this for saas, will update in opensource.